webapp/src/routes/OffersManagement/OffersManagement.js

```diff
@@ -165,26 +162,39 @@ const OffersManagement = () => {
         break
     }
   }
+
+  const [anchorEl, setAnchorEl] = useState(null)
+
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget)
+  };
+
+  const handleClose1 = () => {
```

Comment:
Better not to tag functions with numbers, instead of "handleClose1" in my opinion is better like this "handleCloseMenu" or "handleClose" if it is global.

Reply:
@steph222 take a look at Edgar suggestion

Reply:
That was for the test, I forgot to change it